cmd/kubeseal/main.go

```diff
@@ -53,6 +53,7 @@ var (
 	controllerName = flag.String("controller-name", "sealed-secrets-controller", "Name of sealed-secrets controller.")
 	outputFormat   = flag.StringP("format", "o", "json", "Output format for sealed secret. Either json or yaml")
 	dumpCert       = flag.Bool("fetch-cert", false, "Write certificate to stdout. Useful for later use with --cert")
+	forceEmpty     = flag.Bool("force-empty-data", false, "Allow empty data in the secret object")
```

Comment:
I see that "force" here is used in the common unix pattern of "-f" == "do what I told you", but in this case wouldn't it be clearer to just say `"allow-empty-data"`?

Reply:
Agree. Thank you for the review.